specification/redhatopenshift/resource-manager/Microsoft.RedHatOpenShift/stable/2023-09-04/examples/OpenShiftClusters_Get.json

```diff
@@ -0,0 +1,96 @@
+{
+  "parameters": {
+    "api-version": "2023-09-04",
+    "subscriptionId": "subscriptionId",
+    "resourceGroupName": "resourceGroup",
+    "resourceName": "resourceName"
+  },
+  "responses": {
+    "200": {
+      "body": {
+        "id": "/subscriptions/subscriptionId/resourceGroups/resourceGroup/providers/Microsoft.RedHatOpenShift/OpenShiftClusters/resourceName",
+        "name": "resourceName",
+        "type": "Microsoft.RedHatOpenShift/OpenShiftClusters",
+        "location": "location",
+        "systemData": {
+          "createdBy": "string",
+          "createdByType": "Application",
+          "createdAt": "2020-02-03T01:01:01.1075056Z",
+          "lastModifiedBy": "string",
+          "lastModifiedByType": "Application",
+          "lastModifiedAt": "2020-02-03T01:01:01.1075056Z"
+        },
+        "tags": {
+          "key": "value"
+        },
+        "properties": {
+          "provisioningState": "Succeeded",
+          "clusterProfile": {
+            "domain": "cluster.location.aroapp.io",
+            "version": "4.11.0",
+            "resourceGroupId": "/subscriptions/subscriptionId/resourceGroups/clusterResourceGroup"
+          },
+          "consoleProfile": {
+            "url": "https://console-openshift-console.apps.cluster.location.aroapp.io/"
+          },
+          "servicePrincipalProfile": {
+            "clientId": "clientId"
+          },
+          "networkProfile": {
+            "podCidr": "10.128.0.0/14",
+            "serviceCidr": "172.30.0.0/16",
+            "preconfiguredNSG": "Disabled"
+          },
+          "masterProfile": {
+            "vmSize": "Standard_D8s_v3",
+            "subnetId": "/subscriptions/subscriptionId/resourceGroups/vnetResourceGroup/providers/Microsoft.Network/virtualNetworks/vnet/subnets/master"
+          },
+          "workerProfiles": [
+            {
+              "name": "worker",
+              "vmSize": "Standard_D2s_v3",
+              "diskSizeGB": 128,
+              "subnetId": "/subscriptions/subscriptionId/resourceGroups/vnetResourceGroup/providers/Microsoft.Network/virtualNetworks/vnet/subnets/worker",
+              "count": 3
+            }
+          ],
+          "workerProfilesStatus": [
```

Comment:
>"workerProfilesStatus": [ [](http://example.com/codeflow?start=10&length=25)

What is the difference between workerProfiles and workerProfilesStatus here?

Reply:
workerProfiles filed holds the initial input passed by the client and workerProfilesStatus holds the enriched worker profile data after the ARO cluster provisioning state is comepleted.

Reply:
That is not at all clear from its name. First, being an array, the name should be plural and second, it should say something like activeWorkerProfiles or provisionedWorkerProfiles.

Reply:
updated to activeWorkerProfiles